src/connectors/query-rules/connectQueryRules.ts

```diff
@@ -18,6 +18,10 @@ import {
   NumericRefinement as InternalNumericRefinement,
 } from '../../lib/utils/getRefinements';
 
+type HelperChangeEvent = {
```

Comment:
Since we'll reuse this type we can store it in [instantsearch.ts](https://github.com/algolia/instantsearch.js/blob/6b64d6b0dd2153b5f40fb0121e3a19224c011420/src/types/instantsearch.ts).

Reply:
https://github.com/algolia/instantsearch.js/pull/3796/commits/d0ac107ccea87dd53238b5210e719a2e77a6265c